torch/csrc/jit/passes/dead_code_elimination.cpp

```diff
@@ -147,15 +183,23 @@ class DeadCodeEliminator {
 
     for (auto it = block->nodes().rbegin(); it != block->nodes().rend(); ++it) {
       auto node = *it;
-      for (auto subBlock : node->blocks()) {
-        mark(subBlock);
+      if (node->kind() == prim::Loop) {
```

Comment:
Do we need to check for `c10::onnx::loop` here like we do in the other place?

Reply:
onnx is purely functional and has no aliasing, so we wouldn't need the special behavior